python/GafferUI/ShufflePlugValueWidget.py

```diff
@@ -167,7 +167,7 @@ def setReadOnly( self, readOnly ) :
 		GafferUI.PlugValueWidget.setReadOnly( self, readOnly )
 		self.__plugLayout.setReadOnly( readOnly )
 
-	def childPlugValueWidget( self, childPlug, lazy=True ) :
+	def childPlugValueWidget( self, childPlug ) :
 
 		return self.__plugLayout.plugValueWidget( childPlug, lazy )
```

Comment:
This one snuck under the net 🏃  🐅

Reply:
Thanks, good catch - 9e8d963. Also did one more grep to check I hadn't missed any others.